paymaster/src/rpcMethods.ts

```diff
@@ -0,0 +1,54 @@
+import { BigNumberish, ethers } from 'ethers';
```

Comment:
I could switch to viem later if I wanted to, right?

Reply:
Yes, that shouldn't be a problem